mpeg2/src/interleaving_muxer.rs

```diff
@@ -0,0 +1,189 @@
+use super::EncodeError;
+use crate::muxer::{Muxer, Packet, Stream, StreamConfig};
+use alloc::vec::Vec;
+use core2::io::Write;
+use std::collections::VecDeque;
+use std::time::Duration;
+
+pub struct InterleavingMuxer<W> {
+    muxer: Muxer<W>,
+    max_buffer_duration: u64,
```

Comment:
Can we document the units here somehow? A comment would be alright, but I would probably just put it in the field name:

```suggestion
    max_buffer_duration_90khz: u64,
```

Reply:
👍